scaphandre/README.md

```diff
@@ -0,0 +1,68 @@
+# Agent Check: Scaphandre
+
+## Overview
+
+This check monitors [Scaphandre][1].
```

Comment:
Please add information about what Scaphandre is/does, and why a customer would want to use it.

Reply:
Added a little description about scaphandre.